devicon.json

```diff
@@ -8321,4 +8339,5 @@
         "color": "#F04E98",
         "aliases": []
     }
+    
```

Comment:
```suggestion
```


__
[![sema-logo](https://app.semasoftware.com/img/sema-tray-logo.gif)](https://semasoftware.com/gh) &nbsp;**Summary:** :hammer_and_wrench: This code needs a fix&nbsp; | &nbsp;**Tags:** Inelegant